sandpiper/processes/wps_evaluate_rule.py

```diff
@@ -98,7 +106,15 @@ def _handler(self, request, response):
             process_step="process",
         )
 
-        truth_value = evaluate_rule(rule, rule_getter, variable_getter)
+        try:
+            truth_value = evaluate_rule(rule, rule_getter, variable_getter)
+        except NotImplementedError as e:
+            raise ProcessError(
+                f"{type(e).__name__}: Unable to process expression "
+                "because it contains invalid characters"
```

Comment:
What is the default error message from `p2a` for this not implemented error? Could we just use that?

Reply:
it is `Unable to process expression {}".format(expression)` and usually the expression's have a wide variety of chars which won't work with ProcessError.